cmd/nerdctl/container/container_run_test.go

```diff
@@ -1103,3 +1103,31 @@ func TestCleanupFIFOs(t *testing.T) {
 	}
 	testCase.Run(t)
 }
+
+func TestContainerLiveRestore(t *testing.T) {
+	if runtime.GOOS != "linux" {
+		t.Skip("test only support on linux")
+	}
+	if rootlessutil.IsRootless() {
+		t.Skip("containerd restart need root")
+	}
+	base := testutil.NewBase(t)
+	containerName := testutil.Identifier(t)
+	teardown := func() {
+		base.Cmd("rm", "-f", containerName).Run()
+	}
+	defer teardown()
+	teardown()
+	base.Cmd("run", "-d", "--name", containerName, testutil.CommonImage, "top").AssertOK()
+
+	inspectedContainer := base.InspectContainer(containerName)
+	pid := inspectedContainer.State.Pid
+
+	restartCmd := exec.Command("service", "containerd", "restart")
```

Comment:
`service` command is dead in favor of `systemctl`.

See https://github.com/containerd/nerdctl/blob/92aa908970ea1f054e00219b0a98859687e0ba33/cmd/nerdctl/container/container_run_restart_linux_test.go#L35